akka-docs/src/main/paradox/scala/persistence.md

```diff
@@ -1117,17 +1132,31 @@ In order to help developers build correct and high quality storage plugins, we p
 
 The TCK is usable from Java as well as Scala projects. For @scala[Scala]@java[Java] you need to include the akka-persistence-tck dependency:
```

Comment:
Notice that the note is "smart" on the Java page it'll say Java and for the Scala one it'll say Scala :) 
`@scala[Scala]@java[Java]` cc @sebastianharko  :)